### ISSUE TITLE
"You are not connected to any peers. " after Updated to the latest Parity v1.9.2. What to do?

### ISSUE THREAD

**Issue by MichaelA2014** [paritytech/parity#7937]
_Before filing a new issue, please **provide the following information**._

> I'm running:
>
> - **Which Parity version?**: Parity v1.9.2-20180201 beta

> - **Which operating system?**: Windows 7 x64
> - **How installed?**: via installer windows installer from github
> - **Are you fully synchronized?**: I was before installed the v1.9.2
> - **Which network are you connected to?**: ethereum 
> - **Did you try to restart the node?**: yes

I used to run 1.8.6 and everything worked like a charm. This morning I updated to the v1.9.2 and now when I open parity I get the "You are not connected to any peers. There is most likely some network issue. Fix connectivity." error.
What can I do?  I tried to uninstall v1.9.2 and install older version - the outcome is the same. I get this error. I know there is no problem with my network because I can access everything. I also opened firewall for parity.
What can I do to overcome this ?

Thank you

**Comment by 5chdn** [comment]
Does this issue persist if you leave your node running for 1+ hours?

**Comment by MichaelA2014** [comment]
I left it alone for half a day and it was just sitting there doing nothing - not connecting.
I ended up removing [network] section from my custom config file that used to work great before update. 
I also removed Ethereum folder (backed up KEYS folder first)
Then I removed the DB ( I have DB path set to SSD) then re-installed 1.8.7 stable then imported keys back in and it works again now.

**Comment by 5chdn** [comment]
Could you try to upgrade to 1.9.2 and tell me if this behavior returns?

**Comment by MichaelA2014** [comment]
Sure. I am going to try now and report back here.

**Comment by MichaelA2014** [comment]
I installed 1.9.2 and it seems to be running fine right now.
It's doing sync now. However in a logs I get "permission denied , error 5" Is it normal?

![2018-02-19_9-02-12](https://user-images.githubusercontent.com/15806130/36384185-086e2160-1554-11e8-9f1f-0ada0c83588e.png)

**Comment by Creamers158** [comment]
I confirm both above. Now parity.exe hugs >50% cpu continuesly and syncing takes ages. Just my few cents.

**Comment by 5chdn** [comment]
Where is your parity data dir? Is it an external drive?

**Comment by MichaelA2014** [comment]
@5chdn If you mean me then my parity data is on the system drive. 

The database, however, is on the ssd drive.

**Comment by MichaelA2014** [comment]
The file system is NTFS. The permission is read/write allowed for all users

**Comment by ChittaMadhobi** [comment]
This is what I did (Windows 10)::
1. Deleted the old-versions completely (installation directory and from c:/users/comname/AppData/Roaming/Parity/).
2. Downloaded Parity installer  parity_1.9.7_windows_installer.exe
3. Re-installed.

The thing is: Parity automatically started syncing (I did not start like I used  to do before). Now it says (on browser) that 14% synced and 20/25 peers connected but do not know how to get the address, or get back to my older account (ropsten). 

I wish, there was a little more documentation somewhere. Any help :)?
